core/src/lib.rs

```diff
@@ -7,18 +7,18 @@ extern crate alloc;
 pub mod chiplets;
 pub mod errors;
 
-pub use ::crypto::{Word, EMPTY_WORD, ONE, WORD_SIZE, ZERO};
+pub use ::miden_crypto::{Word, EMPTY_WORD, ONE, WORD_SIZE, ZERO};
 pub mod crypto {
     pub mod merkle {
-        pub use ::crypto::merkle::{
+        pub use ::miden_crypto::merkle::{
             DefaultMerkleStore, EmptySubtreeRoots, InnerNodeInfo, MerkleError, MerklePath,
             MerkleStore, MerkleTree, Mmr, MmrPeaks, NodeIndex, PartialMerkleTree,
             RecordingMerkleStore, SimpleSmt, StoreNode, TieredSmt,
         };
     }
 
     pub mod hash {
-        pub use ::crypto::hash::{
+        pub use ::miden_crypto::hash::{
             blake::{Blake3Digest, Blake3_160, Blake3_192, Blake3_256},
             rpo::{Rpo256, RpoDigest},
             ElementHasher, Hasher,
```

Comment:
Similar comment to the above: we probably don't need `::`  the front of `miden-crypto`.

Reply:
Makes sense!